platform/tezos/model.go

```diff
@@ -48,3 +48,17 @@ type Address struct {
 type Head struct {
 	Level int64 `json:"level"`
 }
+
+type DelegateOptions struct {
+	Setable bool `json:"setable"`
+}
+
+type Account struct {
+	Name          Address         `json:"name"`
+	Manager       Address         `json:"address"`
+	Delegate      DelegateOptions `json:"delegate"`
+	Balance       string          `json:"balance"`
```

Comment:
can you only include what's needed? to keep models light

Reply:
I believe we should describe models to allow us to do easy integrations in the future. Each time you want to know what this API returns you just have to pick in the sources and check. Other way you will be needed to have some pain finding api to trigger, fill it with referrer header to make them work etc.

Reply:
in blockchain space most likely your code will break because of the changes you don't expect and to make it easier to maintain many platforms it's easier to keep only what's needed